packages/rust/build.sh

```diff
@@ -32,17 +37,18 @@ termux_step_configure() {
 	export ${env_host}_OPENSSL_DIR=$TERMUX_PREFIX
 	export X86_64_UNKNOWN_LINUX_GNU_OPENSSL_LIB_DIR=/usr/lib/x86_64-linux-gnu
 	export X86_64_UNKNOWN_LINUX_GNU_OPENSSL_INCLUDE_DIR=/usr/include
+	export PKG_CONFIG_ALLOW_CROSS=1
 	# for backtrace-sys
 	export CC_x86_64_unknown_linux_gnu=gcc
 	export CFLAGS_x86_64_unknown_linux_gnu="-O2"
-	unset CC CXX CPP LD CFLAGS CXXFLAGS CPPFLAGS LDFLAGS PKG_CONFIG
+	unset CC CXX CPP LD CFLAGS CXXFLAGS CPPFLAGS LDFLAGS PKG_CONFIG AR
 }
 
 termux_step_make() {
 	$TERMUX_PKG_SRCDIR/x.py dist \
 		--host $CARGO_TARGET_NAME \
 		--target $CARGO_TARGET_NAME \
-		--target wasm32-unknown-unknown
+		--target wasm32-unknown-unknown || bash
```

Comment:
Yes for random reasons on new versions so I should probably remove that. But dealing with failures is really painful compiling rustc since it takes ages. This is me trging to deal with this late at night kind of unproffesional...